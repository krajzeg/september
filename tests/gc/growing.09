desired_length := 2000
root := Object()
count := 0
object := root

while (count < desired_length) {
	object::next = Object()
	object = object.next	
	count = count + 1
}

print("Successfully allocated a", desired_length, "element linked list.")
