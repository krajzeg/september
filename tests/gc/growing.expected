Successfully allocated a 2000 element linked list.
